arviz/plots/elpdplot.py

```diff
@@ -57,11 +57,11 @@ def plot_elpd(
     ax: axes, optional
         Matplotlib axes or bokeh figures.
     ic : str, optional
-        Information Criterion (WAIC or LOO) used to compare models. Defaults to
+        Information Criterion (PSIS-LOO, WAIC) used to compare models. Defaults to
```

Comment:
maybe something like this. Just to avoid people try passing the string `psis-loo`

Information Criterion (PSIS-LOO ``loo`` or WAIC ``waic``) used to compare models

Reply:
I like that format, let's change these